src/components/pages/AddHouse.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+// import { Navigate } from 'react-router';
```

Comment:
- [ ] Please, consider deleting this old piece of commented code to improve the cleanliness of the repo. 👍